Plugin not running if plugin_name is given from CLI

<!--- Provide a general, concise summary of the issue in the Title above -->
Plugin is not running if I gave plugin name instead of plugin code with -o argument through CLI.

## Expected Behavior
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->
OWTF should run the plugin with plugin name also if given from the CLI.

## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->
If plugin name is given with -o argument then it doesn't run

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, or ideas how to implement the addition or change -->
IMO in `GetAll()` function ( [here](https://github.com/owtf/owtf/blob/develop/framework/db/plugin_manager.py#L215) ) criteria is not checked if it contains plugin name or not. So, need to fix that only I guess.

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to reproduce this bug.-->
<!--- Include code, screenshots or anything we may find helpful to reproduce the issue, if relevant -->
Run the following command and see the terminal 
`./owtf.py -o Old_Backup_and_Unreferenced_Files https://example.org`

## Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->
Plugins will be able to run from their names also.

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Installation method used: automatic
* Version/branch used:develop
* Operating System and version (like Kali, Debian, ArchLinux, etc): kali
* pip/setuptools version: 8.0.3
* sudo access: yes

@DoomTaper @delta24 When I am trying to run plugin from target page I am getting following error.

![screenshot from 2016-05-05 18 51 23](https://cloud.githubusercontent.com/assets/6971456/15044461/66ca0334-12f2-11e6-8832-fecc1031411a.png)

```
[ERROR] [2016-05-05 18:47:09,002] [File 'web.py', line 1811, in log_request] - 500 POST /api/worklist/ (127.0.0.1) 3.05ms
[ERROR] [2016-05-05 18:47:09,005] [File 'web.py', line 1407, in log_exception] - Uncaught exception POST /api/worklist/ (127.0.0.1)
HTTPServerRequest(protocol='http', host='127.0.0.1:8009', method='POST', uri='/api/worklist/', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Referer': 'http://127.0.0.1:8009/ui/targets/16', 'Content-Length': '67', 'Accept-Language': 'en-US,en;q=0.5', 'Accept-Encoding': 'gzip, deflate', 'Host': '127.0.0.1:8009', 'Accept': '*/*', 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:38.0) Gecko/20100101 Firefox/38.0 Iceweasel/38.8.0', 'Connection': 'keep-alive', 'X-Requested-With': 'XMLHttpRequest', 'Pragma': 'no-cache', 'Cache-Control': 'no-cache', 'Cookie': 'csrftoken=4v4ATR97YOyDBLmgcdQiXoX6ZtKBsKoC; __utma=96992031.1135433685.1445873663.1445873663.1446653973.2; _ga=GA1.1.1135433685.1445873663', 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8'})
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1332, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/home/darknight/projects/owtf/framework/interface/api_handlers.py", line 624, in post
    plugin_list = self.get_component("db_plugin").GetAll(filter_data)
  File "/home/darknight/projects/owtf/framework/db/plugin_manager.py", line 227, in GetAll
    Criteria["code"] = self.PluginNametoCode(Criteria["code"])
  File "/home/darknight/projects/owtf/framework/db/plugin_manager.py", line 222, in PluginNametoCode
    codes[count] = str(code[0])
TypeError: 'NoneType' object has no attribute '__getitem__'
```

--- a/framework/db/plugin_manager.py
+++ b/framework/db/plugin_manager.py
@@ -214,17 +214,7 @@ class PluginDB(BaseComponent, DBPluginInterface):
                 query = query.filter(models.Plugin.name.in_(criteria["name"]))
         return query.order_by(models.TestGroup.priority.desc())
 
-    def PluginNametoCode(self, codes):
-        query = self.db.session.query(models.Plugin.code)
-        for count, name in enumerate(codes):
-            if "OWTF-" not in name:
-                code = query.filter(models.Plugin.name == name).first()
-                codes[count] = str(code[0])
-        return(codes)
-
     def GetAll(self, Criteria={}):
-        if Criteria.has_key("code"):
-            Criteria["code"] = self.PluginNametoCode(Criteria["code"])
         query = self.GenerateQueryUsingSession(Criteria)
         plugin_obj_list = query.all()
         return(self.DerivePluginDicts(plugin_obj_list))

